src/app/problems/problem-view/problem-view.component.html

```diff
@@ -3,18 +3,19 @@
         <div class="question-header">
             <h1 class="tui-text_h3 tui-space_vertical-10">{{ uqj?.question?.type_name | titlecase }}</h1>
             <tui-badge
-                class="tui-space_left-4"
-                [status]="uqj?.status | tuiStatus"
-                [value]="uqj?.status || 'No Status'"
+                    class="tui-space_left-4"
```

Comment:
These indents are wrong. I think you have to fix the default tab size in webstorm.

Reply:
What's it supposed to be? I've never changed it...